.github/workflows/screenshots.yml

```
@@ -15,6 +15,8 @@ jobs:
 
     env:
       PERCY_PARALLEL_NONCE: ${{ github.run_id }}-${{ github.run_attempt }}
+      PERCY_TARGET_BRANCH: ${{ github.base_ref }}
+      PERCY_TARGET_COMMIT: ${{ github.event.pull_request.base.sha }}
```

Just in case, can we pair these two together?

Either pick both the shorthand or the longhand then we're all set (if it works!)

```suggestion
      PERCY_TARGET_BRANCH: ${{ github.event.pull_request.base.ref }}
      PERCY_TARGET_COMMIT: ${{ github.event.pull_request.base.sha }}
```